src/components/wallet/earn/ModalDepositFunds.vue

```diff
@@ -362,16 +362,26 @@ export default class ModalDepositFunds extends Vue {
         })
         return isSigned
     }
+
+    get platformUnlocked(): BN {
+        return this.$store.getters['Assets/walletPlatformBalanceUnlocked']
+    }
+
     get interestRateBase(): number {
         return 365 * 24 * 60 * 60
     }
+
     get currentAmount(): string {
         return bnToBig(this.amt, 9).toString()
     }
+
     get maxLeftToDeposit(): BN {
         if (!this.offer.totalMaxAmount.isZero()) {
             return this.offer.totalMaxAmount.sub(this.offer.depositedAmount)
         }
+        if (this.offer.totalMaxRewardAmount.isZero()) {
+            return this.platformUnlocked.sub(ava.PChain().getTxFee())
```

Comment:
can you explain this change?

Reply:
The maxLeftToDeposit() function calculates how much more can be deposited to an offer. and this condition means that if the offer has no limit, then the maximum you can deposit is the total unlocked platform tokens minus the transaction fee required for the transaction.